.github/README_es.md

```diff
@@ -0,0 +1,134 @@
+<h1 align="center">
+    <a href="https://github.com/Shpota/goxygen/tree/master/.github/README.md">
+        <img height="25px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/gb.svg">
+    </a>
+    <a href="https://github.com/Shpota/goxygen/tree/master/.github/README_zh.md">
+        <img height="20px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/cn.svg">
+    </a>
+    <a href="https://github.com/Shpota/goxygen/tree/master/.github/README_ua.md">
+        <img height="20px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/ua.svg">
+    </a>
+    <a href="https://github.com/Shpota/goxygen/tree/master/.github/README_ru.md">
+        <img height="20px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/ru.svg">
+    </a>
+    <a href="https://github.com/Shpota/goxygen/tree/master/.github/README_ko.md">
+        <img height="20px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/kr.svg">
+    </a>
+    <a href="https://github.com/Shpota/goxygen/tree/master/.github/README_pt-br.md">
+        <img height="20px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/br.svg">
+    </a>
+    <a href="https://github.com/Shpota/goxygen/tree/master/.github/README_by.md">
+        <img height="20px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/by.svg">
+    </a>
+    <a href="https://github.com/Shpota/goxygen/tree/master/.github/README_fr.md">
+        <img height="20px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/fr.svg">
+    </a>
+    <a href="https://github.com/Shpota/goxygen/tree/master/.github/README_es.md">
+        <img height="20px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/es.svg">
```

Comment:
Note, Spanish flag should be larger (25px) when the user browses the Spanish version.
```suggestion
        <img height="25px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/es.svg">
```